pagopa/io-app
# chore: [IOBP-140] Fix fruition and onboarding status dates in IDPay initiative details screen

## Short description
This PR fixes the fruition and onboarding dates in IDPay initiative details screen.

https://github.com/pagopa/io-app/assets/6160324/ab7501ec-d043-485a-8d2d-e09bc2c260c7


## List of changes proposed in this pull request
- Updated IDPay API specs in `package.json`
- Added store, actions and saga for IDPay onboarding status
- Added onboarding date in initiative details screen

## How to test
With the `io-dev-server`, navigate to the details screen of an IDPay initiative. Check that fruition dates and onboarding date are correct.
